spec.html

```diff
@@ -26455,9 +26555,16 @@ <h1>Properties of Number Instances</h1>
   <!-- es6num="20.2" -->
   <emu-clause id="sec-math-object">
     <h1>The Math Object</h1>
-    <p>The Math object is the <dfn>%Math%</dfn> intrinsic object and the initial value of the `Math` property of the global object. The Math object is a single ordinary object.</p>
-    <p>The value of the [[Prototype]] internal slot of the Math object is the intrinsic object %ObjectPrototype%.</p>
-    <p>The Math object is not a function object. It does not have a [[Construct]] internal method; it is not possible to use the Math object as a constructor with the `new` operator. The Math object also does not have a [[Call]] internal method; it is not possible to invoke the Math object as a function.</p>
+    <p>The Math object:</p>
+    <ul>
+      <li>is the intrinsic object <dfn>%Math%</dfn>.</li>
+      <li>is the initial value of the `Math` property of the global object.</li>
+      <li>is an ordinary object.</li>
+      <li>has a [[Prototype]] internal slot whose value is the intrinsic object %ObjectPrototype%.</li>
+      <li>is not a function object.</li>
```

Comment:
I wonder if it would be helpful to establish a definition for objects like Math and JSON that are "namespace objects", so the boilerplate on both could be reduced?

Reply:
Note that the spec already has something called a "module namespace object", so introducing the (unrelated) term "namespace object" might be confusing.

We could reduce some boilerplate if we simply acknowledged that "is not a function object" implies "does not have [[Call]] or [[Construct]]". (If we wanted a term for auto-linking, "non-function object" would do.)

Also, assuming it's true that every non-function object necessarily "does not have an [[ECMAScriptCode]] internal slot or any other of the internal slots listed in table 27", then we could reduce some more boilerplate by saying that in one place.